makepkg/PKGBUILD

```diff
@@ -25,7 +25,7 @@ pkgver() {
 
 build() {
     cd aurutils
-    make DESTDIR="$pkgdir"
+    make
```

Comment:
intentional?

Reply:
Yeah, it's the same in the AUR PKGBUILD - `DESTDIR` does nothing in the build step.